src/rdkafka_mock_handlers.c

```diff
@@ -146,15 +151,32 @@ static int rd_kafka_mock_handle_Produce(rd_kafka_mock_connection_t *mconn,
                                             resp, mpart->start_offset);
                                 }
                         }
+
+                        if (rkbuf->rkbuf_reqhdr.ApiVersion >= 8) {
+                                /* Response: #RecordErrors
+                                 * TODO: Add support for injecting RecordErrors
+                                 * 0 record errors for now */
+                                rd_kafka_buf_write_arraycnt(resp, 0);
+
+                                /* Response: ErrorMessage */
+                                rd_kafka_buf_write_str(resp, NULL, 0);
+
+                                /* Response: Partition tags */
+                                rd_kafka_buf_write_tags_empty(resp);
```

Comment:
Kept it inside the block since tags were only needed for v8+. But since rd_kafka_buf_write_tags_empty already has a flexver check it can be moved outside. Updated it